### PR TITLE
Add webpack bundle analyzer in admin

### PR DESCRIPTION
**Pull request by mfrachet**
### What does it do?

Adds a new command: `yarn analyze:bundle` in strapi-admin that aims to display a tree map of the bundles and their dependencies

### Why is it needed?

Helping isolating loading bottlenecks, provide information

### What it looks like

Screenshot 2021-01-25 at 09 45 33

### CODE REVIEW

File: packages/strapi-admin/package.json [hunk 1]
```diff
@@ -125,5 +126,8 @@
     "npm": ">=6.0.0"
   },
   "license": "SEE LICENSE IN LICENSE",
-  "gitHead": "231263a3535658bab1e9492c6aaaed8692d62a53"
+  "gitHead": "231263a3535658bab1e9492c6aaaed8692d62a53",
```
Comment (mfrachet) [inline packages/strapi-admin/package.json:129]: ❓ I'm not used to this attributes, what does it aim for?

Comment (alexandrebodin) [reply packages/strapi-admin/package.json:129]: It is used for npm publish. Lerna creates it temporarily on publish & should remove it after release in theory :) 

Haven't investigated why it doesn't reset it yet

**Review by HichamELBSI — APPROVED** [review]
LGTM

**Review by soupette — APPROVED** [review]
LGTM

**Pull request merged**